templates/common/user_tools.html

```diff
@@ -44,6 +44,10 @@ <h2 class="username">
       <ul id="user-actions" class="toolset clear">
 
         $if not relationship['is_self']:
+		  <li>
+		    $if relationship['followby']:
+			  <span class="icon icon-20 icon-followed"></span>Following You
+		  </li>
```

Comment:
You can make another commit on top of this one and push it to the branch. Just 'fix whitespace' or whatever. PRs will track changes to the branch.

You can also do a git commit --amend and force push to the branch, but GitHub's support for that flow is less stable.